src/raftstore/store/metrics.rs

```diff
@@ -214,4 +214,11 @@ lazy_static! {
             "tikv_raftstore_leader_missing",
             "Total number of leader missed region"
         ).unwrap();
+
+    pub static ref INGEST_SST_DURATION_SECONDS: Histogram =
+        register_histogram!(
+            "tikv_snapshot_ingest_sst_duration_seconds",
+            "Bucketed histogram of duration of rocksdb ingestion",
```

Comment:
"Bucketed histogram of rocksdb ingestion durations"

Reply:
ok